test/tests/ver-1kb/ver-1kb.i

```diff
@@ -0,0 +1,211 @@
+nb_segments = 20
+long_segment = '${units 1.25e-5 m}'
+long_total = '${units ${fparse nb_segments * long_segment} m}'
+simulation_time = '${units 10000 s}'
+T = '${units 500 K}'
+R = '${units 8.31446261815324 J/mol/K}' # ideal gas constant from PhysicalConstants.h
+Na = '${units 6.02214076e23 1/mol}' # Avogadro's number from PhysicalConstants.h
+initial_pressure_1 = '${units 1e5 Pa}'
+initial_pressure_2 = '${units 1e-10 Pa}'
+initial_concentration_1 = '${units ${fparse initial_pressure_1 / (R*T)} mol/m^3}'
+initial_concentration_2 = '${units ${fparse initial_pressure_2 / (R*T)} mol/m^3}'
+solubility = '${fparse 1.082e20/Na}' # Henry's law constant
+diffusivity = '${units ${fparse 4.31e-6 * exp(-2818/T)} m^2/s}'
+n_sorption = 1
+unit_scale = 1
+unit_scale_neighbor = 1
+
+[Mesh]
+  [generated]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = '${fparse nb_segments}'
+    xmax = '${fparse long_total}'
+  []
+  [enclosure_1]
+    type = SubdomainBoundingBoxGenerator
+    input = generated
+    block_id = 1
+    bottom_left = '0 0 0'
+    top_right = '${fparse 1/3 * long_total} 0 0'
+  []
+  [enclosure_2]
+    type = SubdomainBoundingBoxGenerator
+    input = enclosure_1
+    block_id = 2
+    bottom_left = '${fparse 1/3 * long_total} 0 0'
+    top_right = '${fparse long_total} 0 0'
+  []
+  [interface]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = enclosure_2
+    primary_block = 1
+    paired_block = 2
+    new_boundary = interface
+  []
```

Comment:
```suggestion
  [breakmesh]
    input = enclosure_2
    type = BreakMeshByBlockGenerator
    block_pairs = '1 2'
    split_interface = true
    add_interface_on_two_sides = true
  []
```
If you look at the exodus predictions, you'll see that the concentration distribution is sharper at the interface with this approach.